### ISSUE TITLE
Python 3.8: NVDA fails to exit properly after silent install on AppVeyor

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#12153]
### Steps to reproduce:
1. Open [AppVeyor build log for the latest Alpha version of NVDA](https://ci.appveyor.com/project/NVAccess/nvda/builds/38169171/)
2. Click artifacts.
3. Download nvda_install.log and inspect it.

### Actual behavior:
Log ends in the following:
```
CRITICAL - watchdog._crashHandler (08:17:59.581) - MainThread (2656):
NVDA crashed! Minidump written to C:\projects\nvda\testOutput\nvda_install.log\..\nvda_crash.dmp
INFO - watchdog._crashHandler (08:17:59.581) - MainThread (2656):
Listing stacks for Python threads:
Python stack for thread 1028 (watchdog):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 932, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "watchdog.pyc", line 112, in _watcher
  File "winKernel.pyc", line 204, in waitForSingleObject

Python stack for thread 2164 (winInputHook):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 932, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "winInputHook.pyc", line 79, in hookThreadFunc

Python stack for thread 2776 (_UIAHandler.UIAHandler.MTAThread):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 932, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "_UIAHandler.pyc", line 329, in MTAThreadFunc
  File "queue.pyc", line 170, in get
  File "threading.pyc", line 302, in wait

Python stack for thread 5540 (braille._BgThread):
  File "threading.pyc", line 890, in _bootstrap
  File "threading.pyc", line 932, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "braille.pyc", line 2193, in func

Python stack for thread 2656 (MainThread):
  File "nvda.pyw", line 264, in <module>
  File "core.pyc", line 567, in main
  File "wx\core.pyc", line 2237, in MainLoop
  File "watchdog.pyc", line 213, in _crashHandler
  File "watchdog.pyc", line 63, in getFormattedStacksForAllThreads

INFO - watchdog._crashHandler (08:17:59.581) - MainThread (2656):
Restarting due to crash

```
While it appears to be harmless i.e NVDA installs correctly and just fails to exit crash is never ideal and since it happens on AppVeyor it may also occur on normal machines.

### Expected behavior:
NVDA should be able to exit normally after an installation.

#### NVDA installed/portable/running from source:
Happens during install from the launcher.

#### NVDA version:
alpha-21882,528d5708

#### Windows version:
Windows Server 2019 
#### Name and version of other software in use when reproducing the issue:
None
#### Other information about your system:
Happens on an AppVeyor VM
#### Does the issue still occur after restarting your computer?
N/A
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Alphas running under Python 3.7 were able to exit normally after an install on AppVeyor
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
N/a
